src/Routes.js

```diff
@@ -16,9 +19,17 @@ class Routes extends React.Component {
         <Switch>
           <Route exact path="/" component={Main} />
           <Route exact path="/Login" component={Login} />
-          <Route exact path="/Product" component={Product} />
+          {/* <Route
+            exact
+            path="/products"
+            component={Product}
+          /> */}
+          <Route exact path="/products" component={Product} />
           <Route exact path="/Payment" component={Payment} />
           <Route exact path="/Signup" component={Signup} />
+          <Route exact path="/Review" component={Review} />
+          <Route exact path="/ReviewWrite" component={ReviewWrite} />
```

Comment:
```suggestion
          <Route exact path="/review" component={Review} />
          <Route exact path="/review/write" component={ReviewWrite} />
```
주소는 대체로 snake_case로 표현하고, review 쓰기 기능 같은 경우에는 review에 속한 write 기능이기 때문에 path parameter로 표현하면 어떨까 싶어요

Reply:
주소값은 _로 표현하는군요 🥺 ... 수정완료 했습니다!